### ISSUE TITLE
The rule "no-value-attribute" not included in released version 0.5.0 of NPM package

### ISSUE THREAD

**Issue by evangalen** [43081j/eslint-plugin-lit#29]
In the released version 0.5.0 of NPM package the "no-value-attribute" rules is not included.

When running `eslint .` on my project, I get the following errors:

```
  1:1  error  Definition for rule 'lit/no-value-attribute' was not found  lit/no-value-attribute
```

When looking at ./node_modules/eslint-plugin-lit/lib/rules it turns out that the rule is not included.

![image](https://user-images.githubusercontent.com/1895520/52943749-1d269e00-336e-11e9-90eb-2f9257014e01.png)

Whereas the 0.5.0 version is indeed installed by NPM:

![image](https://user-images.githubusercontent.com/1895520/52943858-5a8b2b80-336e-11e9-8fd8-82f54bbbbf13.png)

**Comment by 43081j** [comment]
should be fixed by 0.6.0.

let me know if all goes well

**Comment by evangalen** [comment]
Thanks! Bumped to 0.6.0 and now the"no-value-attribute" indeed works.
